Content.Server/GameObjects/Components/Stack/StackComponent.cs

```diff
@@ -111,6 +120,30 @@ public bool AttackBy(AttackByEventArgs eventArgs)
                 var toTransfer = Math.Min(Count, stack.AvailableSpace);
                 Count -= toTransfer;
                 stack.Add(toTransfer);
+
+                var popupPos = eventArgs.ClickLocation;
+                if (popupPos == GridCoordinates.Nullspace)
+                {
+                    popupPos = eventArgs.User.Transform.GridPosition;
+                }
+
+
+                if (toTransfer > 0)
+                {
+                    _sharedNotifyManager.PopupMessage(popupPos, eventArgs.User, "+" + toTransfer.ToString());
+
+                    if (stack.AvailableSpace == 0)
+                    {
+
+                        Timer.Spawn(300, () => _sharedNotifyManager.PopupMessage(popupPos, eventArgs.User, "Stack is full."));
```

Comment:
Maybe make this say "Stack is now full." to indicate more clearly that it wasn't before.